src/store/hdap.js

```diff
@@ -26,7 +27,7 @@ export const useHdapStore = defineStore('hdapStore', () => {
         currentJwt.value = ''
         friendlyUserName.value = ''
         message.value = (showMessage) ? 'Session expired' : ''
-        router.push('/')
+        if (!skipForTests) router.push('/')
```

Comment:
This line is in the wrong place, I realize.

It's only when the user actively chooses to log out that the application should return to the home page. If you're session times out while you're in the middle of an update, you should be able to log in and continue without interruption.

Reply:
Fixed in my copy